Makefile

```diff
@@ -134,6 +157,13 @@ test-integration: manifests generate fmt vet envtest ginkgo ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) --arch=amd64 use $(ENVTEST_K8S_VERSION) -p path)" \
 	$(GINKGO) -v $(INTEGRATION_TARGET)
 
+.PHONY: test-e2e
+test-e2e: manifests generate fmt vet envtest ginkgo kind-delete kind-create e2e-docker image-load
```

Comment:
I don't think we should have `kind-delete` and `kind-create` here, so that users can run the tests on their existing clusters, if needed. Also `image-load` is specific to kind.

Reply:
But we should setup a kind cluster when running e2e tests on cloud infra, or you mean we should do that in test-infra https://github.com/kubernetes/test-infra/pull/27868

Reply:
Yes, we can do it in test-infra.